cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerSetupService.java

```diff
@@ -159,6 +163,22 @@ public Cluster buildCluster(Map<HostGroup, List<InstanceMetaData>> instanceMetaD
         return cluster;
     }
 
+    private String createDataContext(String sdxContext) {
```

Comment:
This flow step is restartable now, would it work if we post the context 2 times? Otherwise, we need to check if we have already done that.